src/main/java/frc/robot/Constants.java

```diff
@@ -238,11 +238,12 @@ public static class ClimberConstants {
     public static final int LEFT_ARM_ID = 20;
     public static final int RIGHT_ARM_ID = 11;
     public static final double CLIMB_PERCENT = -0.4;
-    public static final double CLIMB_POSITION = 7;
-    public static final double[] POSITION_PID = new double[] {0.01, 0, 0};
-    public static final double[] CURRENT_PID = new double[] {0.01, 0, 0};
+    public static final double CLIMB_POSITION = 1;
```

Comment:
With a 200+ gear ratio, isn't this basically the stowed position? Is that where we want to be targeting?

Reply:
is about 2 degrees at the arm, which moves the robot by about an inch